packages/docusaurus/src/client/docusaurus.js

```diff
@@ -55,8 +55,7 @@ const docusaurus = {
       const chunkAssets = window.__chunkMapping[chunkName] || [];
       return arr.concat(chunkAssets);
     }, []);
-    const dedupedChunkAssets = Array.from(new Set(chunkAssetsNeeded));
```

Comment:
Array.from and Set is unnecessary polyfills, We're also looking into < 10 files only and deduping is already done in https://github.com/facebook/docusaurus/blob/245b50f6e0e683cbabb37706381e44f78b2630af/packages/docusaurus/src/client/prefetch.js#L68-L73